src/Model/CommandResponse.php

```diff
@@ -57,6 +58,9 @@ public function getData(): array
 
     public function isSuccessful(): bool
     {
-        return $this->getStatus() === static::STATUS_OK;
+        // Both OK and SKIPPED command responses are marked as successful to allow easy "command did not fail" detection.
```

Comment:
I would put these comments to method comment cause IDE show it when i try to call method.

Reply:
I'm not sure someone as a user of the library should be concerned about this... I'm seeing this more as a internal encapsulated implementation detail - this is why it wrote it inside, so that only someone interested in the implementation would came across it. 

But you think this should not be encapsulated and part of public phpdoc?